### DISCUSSION TITLE
Completing Starstruck Achievement ⭐

### DISCUSSION THREAD

**Discussion by the1Riddle** [Schweinepriester/github-profile-achievements#875] [Show and tell]
Hey there, Folks! 
Today, I've got something special to share with you that is the project I have done.
So could you help me with your star? Thanks you so much, really appreciate it! 🙌 , and to make it even more `interesting`, after giving a star be sure to `comment with your repo link` so that I can star it in exchange. thanks.

[Link to the repo](https://github.com/the1Riddle/binary_trees)

**Comment by Schweinepriester** [comment]
https://github.com/Schweinepriester/github-profile-achievements/discussions/722
